app/components/invite-team-modal/template.hbs

```diff
@@ -17,7 +17,7 @@
             <option selected=true disabled>Select role...</option>
             {{#each roles as |role|}}
               <option selected={{if (eq selected.id role.id) true null}} value="{{role.id}}">
-                {{if role.privileged "(Admin)" ''}} {{role.name}}
+                {{if role.isOwner "(Owner)" ''}} {{role.name}}
```

Comment:
@sandersonet - This was a change in SPD in an attempt to keep the labeling consistent since the new design has the Owner shield label.